src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/S3ProviderConfiguration.kt

```diff
@@ -103,7 +103,7 @@ class S3ProviderConfiguration {
     fun reportingResourceProvider(@Qualifier("basmAmazonS3") client: AmazonS3, @Value("\${BASM_BUCKET_NAME}") bucketName: String): ReportingProvider {
         logger.info("Using AWS S3 resource provider for move.")
         return ReportingProvider {
-            logger.info("Using bucket $bucketName")
+            logger.debug("Using bucket $bucketName")
```

Comment:
So after running an import in pre-prod this was being logged way too many times so reduced the level.